src/ddmd/cond.d

```diff
@@ -326,6 +326,8 @@ extern (C++) final class VersionCondition : DVCondition
             "CRuntime_DigitalMars",
             "CRuntime_Glibc",
             "CRuntime_Microsoft",
+            "CRuntime_Musl",
+            "CRuntime_uClibc",
```

Comment:
I just went with whatever the project calls itself.  I would probably go for the latter if pushed.